framework/src/media/media_player.c

```diff
@@ -323,7 +321,7 @@ int player_worker(void *args)
 	g_pc.pcmout = NULL;
 
 	PLAY_LOCK();
-	g_pc.state = PLAY_STOPPING;
+	g_pc.state = PLAY_ERROR;
```

Comment:
I will fix all status.

Reply:
when do you it~?

Reply:
We need to discuss about status, and I will fix it on next commit.